### ISSUE TITLE
Powering the Jet engine

### ISSUE THREAD

**Issue by End3rdragon1234** [studio-minus/ppg-bugs-and-requests#2860]
### Feature description

When connected to a power source, the jet engine gets stronger and faster much like the rest of the thrusters in game

### Feature purpose

All the other thrusters have it so, why not this one? Also it would be fun to play around with

**Comment by username90001** [comment]
> ### Feature description
> When connected to a power source, the jet engine gets stronger and faster much like the rest of the thrusters in game
> 
> ### Feature purpose
> All the other thrusters have it so, why not this one? Also it would be fun to play around with
> 
> ### Additional context
> _No response_

other engines use fuel? and power to overheat it
jet engine consumes oxygen and heats up oxygen using burning fuel so making jet engine stronger means stop using chemical fuel and use electricity or nuclear reactors

**Comment by End3rdragon1234** [comment]
> > ### Feature description
> > When connected to a power source, the jet engine gets stronger and faster much like the rest of the thrusters in game
> > ### Feature purpose
> > All the other thrusters have it so, why not this one? Also it would be fun to play around with
> > ### Additional context
> > _No response_
> 
> other engines use fuel? and power to overheat it jet engine consumes oxygen and heats up oxygen using burning fuel so making jet engine stronger means stop using chemical fuel and use electricity or nuclear reactors

yeah but it still needs electricity to function no? so increasing power supply will maek it stronger? idk about this stuff maybe im wrong

**Comment by username90001** [comment]
> > > ### Feature description
> > > When connected to a power source, the jet engine gets stronger and faster much like the rest of the thrusters in game
> > > ### Feature purpose
> > > All the other thrusters have it so, why not this one? Also it would be fun to play around with
> > > ### Additional context
> > > _No response_
> > 
> > 
> > other engines use fuel? and power to overheat it jet engine consumes oxygen and heats up oxygen using burning fuel so making jet engine stronger means stop using chemical fuel and use electricity or nuclear reactors
> 
> yeah but it still needs electricity to function no? so increasing power supply will maek it stronger? idk about this stuff maybe im wrong

jet engines are turbopump and it doesnot need energy to functions they need only fuel that will go in that turbopump and using electric pumps is bad idea because they are extremly weak this why turbopumps used even in rockets

**Comment by End3rdragon1234** [comment]
> > > > ### Feature description
> > > > When connected to a power source, the jet engine gets stronger and faster much like the rest of the thrusters in game
> > > > ### Feature purpose
> > > > All the other thrusters have it so, why not this one? Also it would be fun to play around with
> > > > ### Additional context
> > > > _No response_
> > > 
> > > 
> > > other engines use fuel? and power to overheat it jet engine consumes oxygen and heats up oxygen using burning fuel so making jet engine stronger means stop using chemical fuel and use electricity or nuclear reactors
> > 
> > 
> > yeah but it still needs electricity to function no? so increasing power supply will maek it stronger? idk about this stuff maybe im wrong
> 
> jet engines are turbopump and it doesnot need energy to functions they need only fuel that will go in that turbopump and using electric pumps is bad idea because they are extremly weak this why turbopumps used even in rockets

oh okay. would still be cool to make it stronger

**Comment by username90001** [comment]
> > > > > ### Feature description
> > > > > When connected to a power source, the jet engine gets stronger and faster much like the rest of the thrusters in game
> > > > > ### Feature purpose
> > > > > All the other thrusters have it so, why not this one? Also it would be fun to play around with
> > > > > ### Additional context
> > > > > _No response_
> > > > 
> > > > 
> > > > other engines use fuel? and power to overheat it jet engine consumes oxygen and heats up oxygen using burning fuel so making jet engine stronger means stop using chemical fuel and use electricity or nuclear reactors
> > > 
> > > 
> > > yeah but it still needs electricity to function no? so increasing power supply will maek it stronger? idk about this stuff maybe im wrong
> > 
> > 
> > jet engines are turbopump and it doesnot need energy to functions they need only fuel that will go in that turbopump and using electric pumps is bad idea because they are extremly weak this why turbopumps used even in rockets
> 
> oh okay. would still be cool to make it stronger

isnt powered rocket engine and jet engine already strong enough

**Comment by End3rdragon1234** [comment]
> 

nah enough is nvr enough

**Comment by username90001** [comment]
> > 
> 
> nah enough is nvr enough

jet engine can fly entire map in few secounds and speed limit is 5k meters per secound

**Comment by End3rdragon1234** [comment]
> > > 
> > 
> > 
> > nah enough is nvr enough
> 
> jet engine can fly entire map in few secounds and speed limit is 5k meters per secound

make speed limit higher

**Comment by username90001** [comment]
> > > > 
> > > 
> > > 
> > > nah enough is nvr enough
> > 
> > 
> > jet engine can fly entire map in few secounds and speed limit is 5k meters per secound
> 
> make speed limit higher

i dont know about speed limit

**Comment by End3rdragon1234** [comment]
> > > > > 
> > > > 
> > > > 
> > > > nah enough is nvr enough
> > > 
> > > 
> > > jet engine can fly entire map in few secounds and speed limit is 5k meters per secound
> > 
> > 
> > make speed limit higher
> 
> i dont know about speed limit

the max is like 14,800 smth kmh

**Comment by username90001** [comment]
> > > > > > 
> > > > > 
> > > > > 
> > > > > nah enough is nvr enough
> > > > 
> > > > 
> > > > jet engine can fly entire map in few secounds and speed limit is 5k meters per secound
> > > 
> > > 
> > > make speed limit higher
> > 
> > 
> > i dont know about speed limit
> 
> the max is like 14,800 smth kmh

i say 5k meters per secound that 5 kilometers per secound
